src/components/ConfirmRequestDialog.tsx

```diff
@@ -38,25 +39,39 @@ const ConfirmRequestDialog: FunctionComponent<Props> = ({
         <br />
         <strong>{values.reason}</strong>
       </p>
-      <p className="lbh-body">{formatSentence(deliveryMethods)}</p>
 
-      <ul className="lbh-list">
+      <p className="lbh-body">{formatSentence(deliveryMethods)}</p>
+      <ul className="lbh-list govuk-!-margin-top-2">
         <li>
           <strong>{values.resident.name}</strong>
         </li>
-        <li>{values.resident.email}</li>
-        <li>{values.resident.phoneNumber}</li>
+        <li>
+          <strong>{values.resident.email}</strong>
+        </li>
+        <li>
+          <strong>{values.resident.phoneNumber}</strong>
+        </li>
       </ul>
 
       <p className="lbh-body">For the following evidences:</p>
-      <ul className="lbh-list lbh-list--bullet">
+      <ul className="lbh-list lbh-list--bullet govuk-!-margin-top-2">
         <strong>
           {values.documentTypes.map((id) => (
             <li key={id}>{documentTypes.find((dt) => dt.id == id)?.title}</li>
           ))}
         </strong>
       </ul>
 
+      {values.noteToResident.trim() && (
+        <div className="govuk-inset-text lbh-inset-text">
+          <SVGNoteToResident />
+          <strong>Bespoke note to resident</strong>
+          <p className="govuk-!-margin-top-2" style={{ fontStyle: 'italic' }}>
+            {values.noteToResident}
```

Comment:
Might need to trim here as well to show exactly what will be submitted.

Reply:
Added `sanitiseNoteToResident(values.noteToResident)` so that officer can see exactly what will be sent to the resident (i.e. without any whitespace).